frontend/src/app/task.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { TaskModel } from './models/task.model';
+import { NowService } from './now.service';
+import { Page } from './models/page';
+import { UserService } from './user.service';
+import { TaskCommand } from './models/task.command';
+
+function pageParams(pageNumber: number): HttpParams {
+  return new HttpParams().set('page', pageNumber.toString());
+}
+
+@Injectable()
+export class TaskService {
+
+  constructor(private http: HttpClient,
+              private nowService: NowService,
+              private userService: UserService) { }
+
+  listTodo(pageNumber: number): Observable<Page<TaskModel>> {
+    return this.http.get('/api/tasks', {params: pageParams(pageNumber)});
+  }
+
+  listUrgent(pageNumber: number): Observable<Page<TaskModel>> {
+    return this.http.get('/api/tasks', {
+      params: pageParams(pageNumber).set('before', this.nowService.now().add(7, 'd').format('YYYY-MM-DD'))
```

Comment:
Yes, I thought about it. But it's something we can add later, after we have a first feedback on the feature.